contrib/python-digitalocean/template.py

```diff
@@ -0,0 +1,20 @@
+pkgname = "python-digitalocean"
+pkgver = "1.17.0"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-wheel",
+    "python-setuptools",
+]
+depends = ["python-requests", "python-jsonpickle", "python-pyyaml"]
+checkdepends = depends + ["python-pytest", "python-responses"]
+pkgdesc = "Python module to manage Digital Ocean droplets"
+maintainer = "ttyyls <contact@behri.org>"
+license = "LGPL-3.0-or-later"
+url = "https://github.com/koalalorenzo/python-digitalocean"
+source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "9c9c788ae03a088d0c03a9a59ff7ac6c492caadd4942d4fc58795ee859fc228f"
+# failing tests
+options = ["!check"]
```

Comment:
well figure it out

Reply:
some tests were trying to do network calls so i skipped them